promotion/claim.go

```diff
@@ -99,8 +104,14 @@ func (service *Service) ClaimPromotionForWallet(ctx context.Context, promotionID
 	}
 
 	// TODO lookup and return existing claim if exists?
```

Comment:
@evq Should I resolve this TODO / do you know what I need to do?

Reply:
ah! thanks for raising this - opened https://github.com/brave-intl/bat-go/issues/118. let's address in a separate PR